source/reference/mongosync.txt

```diff
@@ -1,9 +1,95 @@
 .. _c2c-mongosync:
 
+.. default-domain:: mongodb
+
+.. program:: mongosync
+
 =============
 ``mongosync``
 =============
 
-.. default-domain:: mongodb
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Synopsis
+--------
+
+The ``mongosync`` binary is the primary process used in Cluster to Cluster Sync.
+It migrates data from one cluster to another and can keep the clusters
+in continuous sync.
+
+This document provides a complete overview of all command-line options.
+
+Options
+-------
+
+Global Options
+~~~~~~~~~~~~~~
+
+.. option:: --cluster0 <URI>
+
+   Sets the URI of the first cluster.  The first cluster can serve as either the
+   source cluster or the destination cluster in the synchronization process.
```

Comment:
Consider linking out to one of the top level connection page. ( I'll add links to the connection types there and this ref anchor: c2c-conn-top-level )

Reply:
Fixed